undertow/src/main/java/org/wildfly/extension/undertow/AjpListenerResourceDefinition.java

```diff
@@ -46,7 +48,14 @@
             .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
             .setAllowExpression(true)
             .build();
-    public static final OptionAttributeDefinition MAX_AJP_PACKET_SIZE = OptionAttributeDefinition.builder("max-ajp-packet-size", UndertowOptions.MAX_AJP_PACKET_SIZE).setMeasurementUnit(MeasurementUnit.BYTES).setAllowNull(true).setAllowExpression(true).build();
+    public static final OptionAttributeDefinition MAX_AJP_PACKET_SIZE = OptionAttributeDefinition
+            .builder("max-ajp-packet-size", UndertowOptions.MAX_AJP_PACKET_SIZE)
+            .setMeasurementUnit(MeasurementUnit.BYTES)
+            .setAllowNull(true)
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(8192))
```

Comment:
some of them are not available to access. I will withdraw this until https://github.com/undertow-io/undertow/pull/450 is merged.